package/yast2.spec

```diff
@@ -185,6 +185,13 @@ if [ -f "/etc/sysctl.d/30-yast.conf" ]; then
     fi
 fi
 
+# fix xlm problems in older control.xml to prevent xml parser failures
```

Comment:
Given that [we have agreed](https://github.com/yast/yast-installation-control/pull/96#issuecomment-623306963) to make `config:type="string"` the default, this change would be unnecessary if we also allowed `config:type="disksize"` to mean `string` for backward compatibility.

Then we would not need to edit /etc files in RPM post scripts, which is INTRUSIVE and error prone. I consider it a bigger problem than continuing to allow `disksize` in the YaST-XML layer.